lldb/source/Plugins/ObjectFile/Minidump/MinidumpFileBuilder.cpp

```diff
@@ -54,6 +54,13 @@ using namespace lldb;
 using namespace lldb_private;
 using namespace llvm::minidump;
 
+// Set of all the stop reasons minidumps will collect.
```

Comment:
@labath Hey Pavel, is there a more elegant way to define a constant set? I want to define these enums in a unified place so our two uses cases don't diverge, but I don't know a better way